notebooks/Airline Delays - Final Writeup

```diff
@@ -224,6 +227,57 @@ ORDER BY origin_aa1_rain_depth"""))
 
 # COMMAND ----------
 
+# MAGIC %md
+# MAGIC #### Weather Correlation Matrix
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC Before we utilize the weather features in our models we wanted to understand if certain variables are highly correlated.  As shown the correlation matrix below the air temperature and dew point temperature have a high correlation above .5.  As a result, we will only include one of these features in the models we build to prevent greater variance in our model coefficients.  
```

Comment:
"to prevent greater variance in our model coefficients" - not sure what this means/if this is correct? Variance over what distribution? I see this as preventing two things:
1. Derivation of OLS regression formulas rely on the assumption that features are independent
2. It makes it easier for the decision trees to learn the features because it has to consider less features in training.

Reply:
Hi Robin, I found this recap of implications of multicollinearity for logistic regression models which highlights that the parameter model estimates will have greater variance if predictor variables are correlated, let me know if this make sense.  I could clarify that I am referring to impacts on the logistic regression model, https://www.tandfonline.com/doi/abs/10.1080/09720502.2010.10700699#:~:text=Multicollinearity%20is%20a%20statistical%20phenomenon,regression%20model%20are%20highly%20correlated.&text=Multicollinearity%20can%20cause%20unstable%20estimates,confidence%20intervals%20and%20hypothesis%20tests.

Reply:
I update the notebook with based on your comments, and committed the changes, do I run another pull request?